src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -22,7 +22,7 @@ export interface RadioGroupProps {
   options: {
     id: string;
     value: string;
-    label: string;
+    label: ReactNode;
```

Comment:
There are cases where we want to allow the user to format the label text (e.g. bold), so this would allow users to pass a node in addition to string.